llvm/lib/Target/AMDGPU/AMDGPUPassRegistry.def

```diff
@@ -65,6 +65,9 @@ FUNCTION_PASS("amdgpu-unify-divergent-exit-nodes",
               AMDGPUUnifyDivergentExitNodesPass())
 FUNCTION_PASS("amdgpu-usenative", AMDGPUUseNativeCallsPass())
 FUNCTION_PASS("si-annotate-control-flow", SIAnnotateControlFlowPass(*static_cast<const GCNTargetMachine *>(this)))
+FUNCTION_PASS("amdgpu-annotate-varying-branch-weights",
```

Comment:
Passes should be sorted alphabetically here

Reply:
Done in 345269b1523b1c2af48dc3798519031bf30d218d.